plugin-template/ios/Plugin.xcodeproj/project.pbxproj

```diff
@@ -421,6 +434,10 @@
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
 				ENABLE_NS_ASSERTIONS = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				FRAMEWORK_SEARCH_PATHS = (
+					"\"${BUILT_PRODUCTS_DIR}/Capacitor\"",
+					"\"${BUILT_PRODUCTS_DIR}/CapacitorCordova\"",
+				);
```

Comment:
why is this needed?
Both are dependencies in the Podfile, so should be able to find them without this

Reply:
I think @ikeith was getting it to build without cocoapods?